From 310284116ba9c726740e1952af875f778676aff1 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:10:37 -0400
Subject: [powerpc] Remove _INIT_GLOBAL(), _STATIC() and _INIT_STATIC()

Message-id: <1410545655-205645-409-git-send-email-dzickus@redhat.com>
Patchwork-id: 94398
O-Subject: [RHEL7 PATCH 408/626] powerpc: Remove _INIT_GLOBAL(), _STATIC() and _INIT_STATIC()
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit a0e971ffb9d9dae3b9892fb548bd2497db758f60
Author: Anton Blanchard <anton@samba.org>
Date:   Tue Feb 4 16:06:25 2014 +1100

    powerpc: Remove _INIT_GLOBAL(), _STATIC() and _INIT_STATIC()

    Now there are no users of _INIT_GLOBAL(), _STATIC() and _INIT_STATIC()
    we can remove them.

    Signed-off-by: Anton Blanchard <anton@samba.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/include/asm/ppc_asm.h b/arch/powerpc/include/asm/ppc_asm.h
index b02ac12..da16314 100644
--- a/arch/powerpc/include/asm/ppc_asm.h
+++ b/arch/powerpc/include/asm/ppc_asm.h
@@ -210,20 +210,6 @@ name: \
  .type GLUE(.,name),@function; \
 GLUE(.,name):
 
-#define _INIT_GLOBAL(name) \
- __REF; \
- .align 2 ; \
- .globl name; \
- .globl GLUE(.,name); \
- .section ".opd","aw"; \
-name: \
- .quad GLUE(.,name); \
- .quad .TOC.@tocbase; \
- .quad 0; \
- .previous; \
- .type GLUE(.,name),@function; \
-GLUE(.,name):
-
 #define _KPROBE(name) \
  .section ".kprobes.text","a"; \
  .align 2 ; \
@@ -238,30 +224,6 @@ name: \
  .type GLUE(.,name),@function; \
 GLUE(.,name):
 
-#define _STATIC(name) \
- .section ".text"; \
- .align 2 ; \
- .section ".opd","aw"; \
-name: \
- .quad GLUE(.,name); \
- .quad .TOC.@tocbase; \
- .quad 0; \
- .previous; \
- .type GLUE(.,name),@function; \
-GLUE(.,name):
-
-#define _INIT_STATIC(name) \
- __REF; \
- .align 2 ; \
- .section ".opd","aw"; \
-name: \
- .quad GLUE(.,name); \
- .quad .TOC.@tocbase; \
- .quad 0; \
- .previous; \
- .type GLUE(.,name),@function; \
-GLUE(.,name):
-
 #else /* 32-bit */
 
 #define _ENTRY(n) \
-- 
1.7.1